src/Chapter1.hs

```diff
@@ -620,8 +643,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n =
+  let last = n `mod` 10
+      sndLast = (n `div` 10) `mod` 10
+  in (last + sndLast)
```

Comment:
No need to put this in parenthesis 🙂 
```suggestion
  in last + sndLast
```

Also, `last` is the name for the standard function, so it is better to name variables in other names 🙂